src/github.com/getlantern/lantern-mobile/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Dockerfile to build an image able to compile flashlight for iOS and Android
+#
+# > make docker
+
+FROM fedora:22
+MAINTAINER "Ulysses Aalto" <uaalto@getlantern.org>
+
+ENV GO_VERSION go1.4.2
+ENV GOROOT_BOOTSTRAP /go1.4
+ENV GOROOT /go
+ENV GOPATH /
+
+ENV PATH $PATH:$GOROOT/bin
+ENV WORKDIR /lantern
+
+# Updating system.
+RUN dnf -y update && dnf clean all
+
+# Requisites for building Go.
+RUN dnf install -y git tar patch gzip curl hostname pcre-devel && dnf clean all
```

Comment:
While building the image I found some logs like this:

```
./test.bash: line 8: which: command not found
```

Can we add the which command here as well?